core/accelogic/src/ZipAccelogic.cxx

```diff
@@ -188,9 +187,10 @@ void R__zipBLAST(int *cxlevels, int *srcsize, char *src, int *tgtsize, char **tg
       // tgt[2] is set for each target buffer above
 
       // Include the 2 extra header byte into the out size.
-      tgt[3] = (char)((out_sizes[tgt_idx] + 2) & 0xff);
-      tgt[4] = (char)((out_sizes[tgt_idx] >> 8) & 0xff);
-      tgt[5] = (char)((out_sizes[tgt_idx] >> 16) & 0xff);
+      size_t out_size = out_sizes[tgt_idx] + 2;
```

Comment:
Is this change cosmetic?

Reply:
No. As you had written it, if out_sizes[tgt_idx] = 255, then tgt[3] = 1, tgt[4] = 0, and tgt[5] = 0, which would be interpreted as having a size of 1. The way I wrote it, tgt[4] would also become 1, so that the true size of 255 + 2 = 257 is preserved.

Reply:
Right.  Thanks!